tox.ini

```diff
@@ -17,5 +22,12 @@ deps =
     -rrequirements_test.txt
     django32: Django>=3.2,<3.3
     django42: Django>=4.2,<4.3
+    postgres: psycopg2-binary>=2.9.6
```

Comment:
There are so many unrelated things in this PR. Why do we need postgres and mariadb here? Even if we have reasons, that's definitely a topic for a separate discussion in a separate PR

Reply:
> Why do we need postgres and mariadb here?

It's used only for tests. 
1. There's been added functionality that could swap model. 
2. This project contain a lot of logic that tied to the model.
3. I wish there were tests that would test the functionality of the database, which are more strict than Sqlite

Reply:
Nothing in the functionality that was added in this PR requires `postgres` or `mariadb` for tests

Reply:
> Nothing in the functionality that was added in this PR requires `postgres` or `mariadb` for tests

Will tests on more databases break anything?

SQLite doesn't enforce `max_length` for text fields. So test `test_registration_id_size` could never fail. 
But yep:
* it could be done in a different PR. 
* Should it done in a different pr? - Well, I don't think that it's required.

Reply:
> As I do think this PR contains significant changes for this repo, it would greatly enhance review-ability if you could extract the non-swappable-related changes into a separate PR, if you would be up for it.

@xtrinch 

Make two PRs:
1. With tests of existed functionality;
2. With swapper functionality and tests for it?

Or do you have a different view of the breakdown of these PRs?

Reply:
No, sounds like a good plan